src/lib/tsc-watch.ts

```diff
@@ -99,12 +99,21 @@ function spawnTsc({ maxNodeMem }: INodeSettings, args: string[]): ChildProcess {
   return spawn('node', nodeArgs);
 }
 
+function echoExit(code: number | null, signal: string | null) {
+  if (signal !== null) {
+    process.kill(process.pid, signal);
+  }
+}
+
 let compilationErrorSinceStart = false;
 const tscProcess = spawnTsc({ maxNodeMem }, args);
-if (!tscProcess.stdout) {
+if (!tscProcess.stderr || !tscProcess.stdout) {
```

Comment:
how about throwing a different error when `stderr` is not available?

Reply:
@gilamran Makes sense with the error message 32f095fc365dc4ad0935c4c3389ea39146a26fa0